clinica/iotools/bids_utils.py

```diff
@@ -333,33 +412,43 @@ def write_scans_tsv(bids_dir, bids_ids, scans_dict):
     from os import path
     import os
     from glob import glob
+    from clinica.utils.stream import cprint
 
     for bids_id in bids_ids:
-        scans_df = pd.DataFrame()
-        bids_id = bids_id.split(os.sep)[-1]
         # Create the file
-        tsv_name = bids_id + "_ses-M00_scans.tsv"
-        # If the file already exists, remove it
-        if os.path.exists(path.join(bids_dir, bids_id, 'ses-M00', tsv_name)):
-            os.remove(path.join(bids_dir, bids_id, 'ses-M00', tsv_name))
-
-        mod_available = glob(path.join(bids_dir, bids_id, 'ses-M00', '*'))
-        for mod in mod_available:
-            mod_name = os.path.basename(mod)
-            files = glob(path.join(mod, '*'))
-            for file in files:
-                file_name = os.path.basename(file)
-                if mod_name == "anat" or mod_name == "dwi" or mod_name == "func":
-                    f_type = 'T1/DWI/fMRI/FMAP'
-                elif mod_name == 'pet':
-                    f_type = 'FDG'
-
-                row_to_append = pd.DataFrame(scans_dict[bids_id][f_type], index=[0])
-                # Insert the column filename as first value
-                row_to_append.insert(0, 'filename', path.join(mod_name, file_name))
-                scans_df = scans_df.append(row_to_append)
-
-            scans_df.to_csv(path.join(bids_dir, bids_id, 'ses-M00', tsv_name), sep='\t', index=False, encoding='utf8')
+        sessions_paths = glob(path.join(bids_dir, bids_id, 'ses-*'))
+        for session_path in sessions_paths:
+            scans_df = pd.DataFrame()
+            session_name = session_path.split(os.sep)[-1]
+            tsv_name = bids_id + '_' + session_name + "_scans.tsv"
```

Comment:
Small questions:

- is `bids_id` refers to `<participant_id>` e.g. `sub-CLNC01`?
- is `session_name ` refers to `<session_id>` e.g. `ses-M00`?

If so, I would advice to change var names so that it improves readability and harmonization.

Reply:
These variables are always named like this in all data converters... Does this mean I have to change them everywhere?

Reply:
I see. To keep this PR easy to read, we will then not change that.

Can you just tell me if `bids_id` refers to `<participant_id>` (e.g. `sub-CLNC01`) and `session_name ` to `<session_id>` (e.g. `ses-M00`)?

I am thinking of doing a PR to harmonize this in order to have the same wording in Clinica.

Reply:
`bids-id` indeed refers to `participant_id` and `session_name` to `session_id`!